app/controllers/concerns/blacklight/search_context.rb

```diff
@@ -111,9 +111,8 @@ def setup_next_and_previous_documents
       response, documents = search_service.previous_and_next_documents_for_search index, search_state.reset(current_search_session.query_params).to_hash
 
       search_session['total'] = response.total
-      @search_context_response = response
-      @previous_document = documents.first
-      @next_document = documents.last
+      search_session['prev'] = documents.first
```

Comment:
I'm a little surprised this works. Do you know if we are creating serialized blobs in the session store every time we do this?

Reply:
Crap, I didn't realize this was the object in session. I'm going to rework this.